### ISSUE TITLE
Ground Zeroes unique characters

### ISSUE THREAD

**Issue by Aladore384** [unknown321/mgswaifus#2]
Hi,

Just an FYI, unique characters from Ground Zeroes are listed under category **Mission -2: No mission attached (sideops?) (27)**. You can find the list here:

https://metalgear.fandom.com/wiki/Mother_Base_Bonus_Staff

Or I can copy/paste for convenience sake:

12 standard enemy combatants:
4 in the Main Mission (Ochre Chameleon, Grizzly Hedgehog, Hungry Crocodile, Wild Harrier)
2 in Eliminate the Renegade Threat (Blue Chameleon, Midnight Mastiff)
3 in Classified Intel Acquisition Frigid Mongoose, Hunting Stallion, Assassin Harrier)
3 in Destroy the Anti-Air Emplacements (Night Tree Frog, Pirate Capybara, Roaring Capybara) 

Prisoner 12282 (Destroy the Anti-Air Emplacements) (Bitter Centipede)
Escaped prisoner in the main mission (Gray Wallaby)
Glaz (Eliminate the Renegade Threat) (Eye)
Palitz (Eliminate the Renegade Threat) (Finger)
Hideo (Intel Operative Rescue) (Hideo)
Undercover JCS agent (Classified Intel Acquisition) (Mad Wallaby)
Bald Marine (Classified Intel Acquisition) (Wild Stallion)
Tank soldier (Déjà Vu; when using the Classic Snake or Cyborg Ninja skins) (Crying Sturgeon)

They're all unlocked after certain missions, and all of them will be unlocked by Mission 12.

So basically, you can divide the category **Mission -2: No mission attached (sideops?) (27)** into 3: one for GZ staff, one for side ops Unlucky Dog (Ziang Tan and Dire Armadillo), and one for side ops Legendary Gunsmith (Devil Chameleon, Shining Kangaroo and Gray Mongoose).

The only characters I have yet to pinpoint are Doom Squirrel, Cunning Mongoose, Dancing Mongoose and Blue Armadillo. I think I read somewhere (maybe from you) that Howling Capybara is the chief of child soldiers during Mission 25, and he was never properly implemented.

Aside from this, your list looks accurate. Thank you for sharing it, it's been very useful.

**Comment by unknown321** [comment]
thanks, updated

**Comment by Aladore384** [comment]
Doom Squirrel, Cunning Mongoose, and Dancing Mongoose are prisoners from Unlucky Dogs sideops, just like Dire Armadillo. Still have to figure out who Blue Armadillo is though.

EDIT: Now that I think of it, Blue Armadillo is most likely a dummied Ground Zeroes soldier. His ID and language fit the profile.
